app/frontend/src/components/Datepicker.tsx

```diff
@@ -5,6 +5,16 @@ import {
 } from "@material-ui/pickers";
 import { Control, Controller } from "react-hook-form";
 
+import { dateFormats } from "./DateFormats";
+
+const getLocaleFormat = () => {
+  for (const [key, value] of Object.entries(dateFormats)) {
+    if (key === navigator.language) {
+      return value.toString();
+    }
+  }
+};
+
 interface DatepickerProps {
```

Comment:
`helperText` should be typed as `React.ReactNode` here, the same type as the `helperText` prop in the underlying `KeyboardDatePicker` (you can see it when hovering it over if using VS Code). We should avoid using `any` in our code if possible.